src/js/components/Drop/DropContainer.js

```diff
@@ -53,13 +84,16 @@ const DropContainer = forwardRef(
   ) => {
     const containerTarget = useContext(ContainerTargetContext);
     const theme = useContext(ThemeContext) || defaultProps.theme;
+    // dropOptions was created to preserve backwards compatibility
+    const { drop: dropOptions } = useContext(OptionsContext);
     const portalContext = useContext(PortalContext);
     const portalId = useMemo(() => portalContext.length, [portalContext]);
     const nextPortalContext = useMemo(
       () => [...portalContext, portalId],
       [portalContext, portalId],
     );
     const dropRef = useForwardedRef(ref);
+    const target = dropTarget?.current || dropTarget;
```

Comment:
Why does this need to be defined outside of the useEffect?

Reply:
It do not need to be defined outside.
I followed the recommendations of @jonscottclark here : https://github.com/grommet/grommet/pull/6097#pullrequestreview-1563661513

I just moved it into the useEffect to save some computation.
Yours faithfully,
LCDP